src/management-system-v2/components/process-creation.module.scss

```diff
@@ -1,6 +1,6 @@
 .Checkbox {
   font-size: 12px;
-  // width: 55%;
+  width: 65%;
```

Comment:
Maybe `display: flex` is a better way to have the checkbox and the buttons on different rows.